models/actor.js

```diff
@@ -11,7 +11,8 @@ const ActorSchema = new mongoose.Schema({
   loc: { type: Object, index: '2dsphere' },
   domains: Array,
   contactName: String,
-  distance: Number
+  distance: Number,
+  actions: Array
```

Comment:
Why do you want to keep this?

Reply:
C'est nécessaire pour pouvoir retourner les `actions` depuis les `actors`. Sans cela, quand on fait notre réponse en JSON de notre `actor`, le champ `actions` est neutralisé par mongoose (qui doit considérer que c'est un champ polluant à ne pas retourner dans la sérialisation). C'est ce que j'ai trouvé de mieux et ça me semble compréhensible.